less/index.less

```diff
@@ -1 +1,8 @@
 // Follow the order in the readme
+@import "less/variables.less";
```

Comment:
Here we can do `@import "<filename>.less";` because this index file is already in the `less/` folder 👍